From b63ee7d836952cb29dae02e3d2b748f3ba5c14d7 Mon Sep 17 00:00:00 2001
From: Timo Juhani Lindfors <timo.lindfors@iki.fi>
Date: Wed, 6 Mar 2013 18:10:18 +0200
Subject: [PATCH 1/2] install stap-prep.1 man page

---
 man/Makefile.am |    2 +-
 1 files changed, 1 insertions(+), 1 deletions(-)

diff --git a/man/Makefile.am b/man/Makefile.am
index 0948c26..4ad0209 100644
--- a/man/Makefile.am
+++ b/man/Makefile.am
@@ -19,7 +19,7 @@ install-data-local:
 	done
 
 if BUILD_TRANSLATOR
-man_MANS += stap.1
+man_MANS += stap.1 stap-prep.1
 endif
 
 if HAVE_NSS
-- 
1.7.2.5

