From 5409bb175f827a7a0f2c0f6c527a8ae7e8eac0f2 Mon Sep 17 00:00:00 2001
From: Timo Juhani Lindfors <timo.lindfors@iki.fi>
Date: Mon, 25 Feb 2013 15:41:20 +0200
Subject: [PATCH 2/3] Makefile.am: remove temporary symlinks in distclean

---
 Makefile.am |    1 +
 1 files changed, 1 insertions(+), 0 deletions(-)

diff --git a/Makefile.am b/Makefile.am
index 4753365..e64df4d 100644
--- a/Makefile.am
+++ b/Makefile.am
@@ -285,6 +285,7 @@ clean-local:
 
 distclean-local:
 	rm -rf build-elfutils include-elfutils lib-elfutils
+	rm -f doc/beginners/en-US doc/beginners/build/en-US/testsuite
 
 uninstall-local:
 	rm -rf $(DESTDIR)$(pkgdatadir)
-- 
1.7.2.5

