From 749896e8669451cdffdc556a771931e840c30f06 Mon Sep 17 00:00:00 2001
From: Timo Juhani Lindfors <timo.lindfors@iki.fi>
Date: Wed, 6 Mar 2013 16:48:27 +0200
Subject: [PATCH 4/4] man stap.1: add reference to stap-prep (1)

---
 man/stap.1 |    1 +
 1 files changed, 1 insertions(+), 0 deletions(-)

diff --git a/man/stap.1 b/man/stap.1
index be0aafd..a7ade7b 100644
--- a/man/stap.1
+++ b/man/stap.1
@@ -2109,6 +2109,7 @@ stappaths (7) manual page.
 .IR stapvars (3stap),
 .IR stapex (3stap),
 .IR stap\-server (8),
+.IR stap\-prep (1),
 .IR awk (1),
 .IR gdb (1)
 
-- 
1.7.2.5

