From 9fcd9ef3277beebb2a84c1aeff24e8648037860b Mon Sep 17 00:00:00 2001
From: Timo Juhani Lindfors <timo.lindfors@iki.fi>
Date: Mon, 25 Feb 2013 15:13:05 +0200
Subject: [PATCH 4/4] Makefile.in: regenerate with automake

---
 Makefile.in |    1 +
 1 file changed, 1 insertion(+)

diff --git a/Makefile.in b/Makefile.in
index fa7f8a0..0889712 100644
--- a/Makefile.in
+++ b/Makefile.in
@@ -2095,6 +2095,7 @@ clean-local:
 
 distclean-local:
 	rm -rf build-elfutils include-elfutils lib-elfutils
+	rm -f doc/beginners/en-US doc/beginners/build/en-US/testsuite
 
 uninstall-local:
 	rm -rf $(DESTDIR)$(pkgdatadir)
-- 
1.7.10.4

