debian version patch
Index: systemtap/Makefile.am
===================================================================
--- systemtap.orig/Makefile.am	2011-02-25 01:17:34.534391001 +0530
+++ systemtap/Makefile.am	2011-02-25 01:29:50.434391001 +0530
@@ -82,7 +82,7 @@
 			cp "$(srcdir)/git_version.h" "git_version.h"; \
 		fi; \
 	fi
-	$(GIT_VERSION_CMD) -k -s $(top_srcdir) -o git_version.h
+	sh $(top_srcdir)/debian/debversion.sh $(top_srcdir)/debian/changelog > git_version.h
 	@if test -s "$(srcdir)/git_version.h"; then \
 		if cmp "$(srcdir)/git_version.h" "git_version.h"; then :; \
 		else \
@@ -93,7 +93,7 @@
 	fi
 
 git_version.h:
-	$(srcdir)/git_version.sh -k --srcdir $(srcdir) -o git_version.h
+	sh $(top_srcdir)/debian/debversion.sh $(top_srcdir)/debian/changelog > git_version.h
 
 cscope:
 	cd $(srcdir) && \
