Index: systemtap/configure.ac
===================================================================
--- systemtap.orig/configure.ac	2010-01-21 09:00:30.000000000 +0100
+++ systemtap/configure.ac	2010-01-21 09:04:00.483768663 +0100
@@ -260,18 +260,18 @@
 fi
 AM_CONDITIONAL([XMLTO_STRINGPARAM], [test "$have_xmlto_stringparam" == "yes"])
 
-if test "x${building_refdocs}" == "xyes"; then
-AC_MSG_CHECKING([for xmlto pdf support])
-   xmlto pdf conftest.xml >& /dev/null
-   if test $? == 0; then
-      have_xmlto_pdf="yes"
-      AC_MSG_RESULT([yes])
-   else
-      AC_MSG_RESULT([no])
-      AC_MSG_WARN([Not building reference documentation in PDF format])
-   fi
-   rm -f conftest.pdf
-fi
+#if test "x${building_refdocs}" == "xyes"; then
+#AC_MSG_CHECKING([for xmlto pdf support])
+#   xmlto pdf conftest.xml >& /dev/null
+#   if test $? == 0; then
+#      have_xmlto_pdf="yes"
+#      AC_MSG_RESULT([yes])
+#   else
+#      AC_MSG_RESULT([no])
+#      AC_MSG_WARN([Not building reference documentation in PDF format])
+#   fi
+#   rm -f conftest.pdf
+#fi
 dnl Unfortunately https://bugzilla.redhat.com/show_bug.cgi?id=526273 
 dnl blocks "xmlto pdf" on our docs (but not on the above test).
 dnl So we need to disable this anyway.
