Index: b/dtrace.in
===================================================================
--- a/dtrace.in
+++ b/dtrace.in
@@ -387,7 +387,7 @@ def main():
         cc1 = os.environ.get("CC", "gcc")
         cflags = "-g " + os.environ.get("CFLAGS", "")
         retcode = call(split(cc1) + defines + includes + split(cflags) +
-             ["-fPIC", "-I.", "-I@prefix@/include", "-c", fname, "-o",
+             ["-fPIC", "-I.", "-c", fname, "-o",
               filename + suffix], shell=False)
         if retcode != 0:
             print "\"gcc " + fname + "\" failed"
