Index: systemtap/configure.ac
===================================================================
--- systemtap.orig/configure.ac	2010-05-21 23:13:28.650544125 +0530
+++ systemtap/configure.ac	2010-05-21 23:13:31.038530019 +0530
@@ -409,11 +409,11 @@
     save_LIBS="$LIBS"
     AC_CHECK_LIB(dw, dwfl_module_getsym,[],[
       AC_MSG_ERROR([missing elfutils development headers/libraries (install elfutils-devel, libebl-dev, libdw-dev and/or libebl-devel)])],
-      [-Wl,--start-group -ldw -lebl -Wl,--end-group -lelf])
+      [-ldw -lelf])
     AC_CHECK_LIB(dw, dwarf_getelf,[],[
       AC_MSG_ERROR([elfutils, libdw too old, need 0.126+])],
-      [-Wl,--start-group -ldw -lebl -Wl,--end-group -lelf])
-    stap_LIBS="$stap_LIBS -Wl,--start-group -ldw -lebl -Wl,--end-group -lelf"
+      [-ldw -lelf])
+    stap_LIBS="-ldw -lelf"
     LIBS="$save_LIBS"
   else
     # We built our own and stap_LDFLAGS points at the install.
Index: systemtap/configure
===================================================================
--- systemtap.orig/configure	2010-05-21 23:13:42.130559003 +0530
+++ systemtap/configure	2010-05-21 23:14:09.930038548 +0530
@@ -8290,7 +8290,7 @@
   $as_echo_n "(cached) " >&6
 else
   ac_check_lib_save_LIBS=$LIBS
-LIBS="-ldw -Wl,--start-group -ldw -lebl -Wl,--end-group -lelf $LIBS"
+LIBS="-ldw -Wl,--start-group -ldw -Wl,--end-group -lelf $LIBS"
 cat >conftest.$ac_ext <<_ACEOF
 /* confdefs.h.  */
 _ACEOF
@@ -8370,7 +8370,7 @@
   $as_echo_n "(cached) " >&6
 else
   ac_check_lib_save_LIBS=$LIBS
-LIBS="-ldw -Wl,--start-group -ldw -lebl -Wl,--end-group -lelf $LIBS"
+LIBS="-ldw -Wl,--start-group -ldw -Wl,--end-group -lelf $LIBS"
 cat >conftest.$ac_ext <<_ACEOF
 /* confdefs.h.  */
 _ACEOF
@@ -8443,11 +8443,11 @@
    { (exit 1); exit 1; }; }
 fi
 
-    stap_LIBS="$stap_LIBS -Wl,--start-group -ldw -lebl -Wl,--end-group -lelf"
+    stap_LIBS="$stap_LIBS -Wl,--start-group -ldw -Wl,--end-group -lelf"
     LIBS="$save_LIBS"
   else
     # We built our own and stap_LDFLAGS points at the install.
-    stap_LIBS="$stap_LIBS -Wl,--start-group -ldw -lebl -Wl,--end-group -lelf"
+    stap_LIBS="$stap_LIBS -Wl,--start-group -ldw -Wl,--end-group -lelf"
   fi
 fi
 
