Index: systemtap/configure.ac
===================================================================
--- systemtap.orig/configure.ac	2009-09-11 00:15:50.400587290 +0200
+++ systemtap/configure.ac	2009-09-11 00:17:18.956372518 +0200
@@ -523,12 +523,13 @@
 # userspace might be 32-bit default, but staprun needs to be
 # 64-bit.  See also bug #4037.
 
-processor=`uname -p`
-case "$processor" in
-ppc64) PROCFLAGS=-m64 ;;
-x86_64) PROCFLAGS=-m64 ;;
-*) PROCFLAGS=""
-esac
+#processor=`uname -p`
+#case "$processor" in
+#ppc64) PROCFLAGS=-m64 ;;
+#x86_64) PROCFLAGS=-m64 ;;
+#*) PROCFLAGS=""
+#esac
+PROCFLAGS=""
 AC_SUBST([PROCFLAGS])
 
 # Use tr1/unordered_map if available
