Index: systemtap/doc/SystemTap_Tapset_Reference/Makefile.am
===================================================================
--- systemtap.orig/doc/SystemTap_Tapset_Reference/Makefile.am	2009-09-11 00:15:49.526401205 +0200
+++ systemtap/doc/SystemTap_Tapset_Reference/Makefile.am	2009-09-11 00:17:46.577200975 +0200
@@ -22,6 +22,9 @@
 PDFDOCS = tapsets.pdf
 endif
 
+DOCBOOK_MAN = /usr/share/xml/docbook/stylesheet/docbook-xsl/manpages/docbook.xsl
+DOCBOOK_HTML = /usr/share/xml/docbook/stylesheet/docbook-xsl/html/chunk.xsl
+
 if BUILD_REFDOCS
 all: $(PDFDOCS) stamp-htmldocs stamp-mandocs
 tapsets.xml: docproc $(shell find $(SRCTREE)/tapset -name '*.stp')
@@ -34,7 +37,8 @@
 	fi
 
 stamp-htmldocs: tapsets.xml
-	xmlto html -o tapsets tapsets.xml
+	$(MKDIR_P) tapsets
+	xsltproc -o tapsets/ --nonet $(DOCBOOK_HTML) tapsets.xml
 	touch stamp-htmldocs
 
 # bump up the allocated space so "xmlto pdf" works
@@ -42,7 +46,8 @@
 	env pool_size=2000000 hash_extra=2000000 xmlto pdf tapsets.xml
 
 stamp-mandocs: tapsets.xml
-	xmlto man -o man3 tapsets.xml
+	$(MKDIR_P) man3
+	xsltproc --stringparam man.copyright.section.enabled 0 -o man3/ --nonet $(DOCBOOK_MAN) tapsets.xml
 	touch stamp-mandocs
 
 #FIXME need to figure out where to install things appropriately
