Index: systemtap/stap-authorize-server-cert
===================================================================
--- systemtap.orig/stap-authorize-server-cert	2009-09-10 19:39:11.000000000 +0200
+++ systemtap/stap-authorize-server-cert	2009-09-11 00:19:20.369698414 +0200
@@ -11,7 +11,7 @@
 # later version.
 
 # Initialize the environment
-. `dirname $0`/stap-env
+. /usr/share/systemtap/stap-env
 
 certfile=$1
 certdb=$2
Index: systemtap/stap-authorize-signing-cert
===================================================================
--- systemtap.orig/stap-authorize-signing-cert	2009-09-10 19:39:11.000000000 +0200
+++ systemtap/stap-authorize-signing-cert	2009-09-11 00:19:20.369698414 +0200
@@ -11,7 +11,7 @@
 # later version.
 
 # Initialize the environment
-. `dirname $0`/stap-env
+. /usr/share/systemtap/stap-env
 
 certfile=$1
 certdb=$2
Index: systemtap/stap-client
===================================================================
--- systemtap.orig/stap-client	2009-09-10 19:39:11.000000000 +0200
+++ systemtap/stap-client	2009-09-11 00:19:20.369698414 +0200
@@ -22,7 +22,7 @@
 trap 'ignore_signal' SIGHUP SIGPIPE
 
 # Initialize the environment
-. `dirname $0`/stap-env
+. /usr/share/systemtap/stap-env
 
 #-----------------------------------------------------------------------------
 # Helper functions.
Index: systemtap/stap-env
===================================================================
--- systemtap.orig/stap-env	2009-09-10 19:39:11.000000000 +0200
+++ systemtap/stap-env	2009-09-11 00:19:20.369698414 +0200
@@ -1,5 +1,3 @@
-#!/bin/bash
-
 # Generate a certificate for the systemtap server and add it to the
 # database of trusted servers for the client.
 #
Index: systemtap/stap-find-or-start-server
===================================================================
--- systemtap.orig/stap-find-or-start-server	2009-09-10 19:39:11.000000000 +0200
+++ systemtap/stap-find-or-start-server	2009-09-11 00:19:20.369698414 +0200
@@ -18,7 +18,7 @@
 # Otherwise, it echoes -1
 
 # Initialize the environment
-. `dirname $0`/stap-env
+. /usr/share/systemtap/stap-env
 
 # Is there a server available?
 ${stap_exec_prefix}stap-find-servers >/dev/null 2>&1 && echo 0 && exit 0
Index: systemtap/stap-find-servers
===================================================================
--- systemtap.orig/stap-find-servers	2009-09-10 19:39:11.000000000 +0200
+++ systemtap/stap-find-servers	2009-09-11 00:19:20.369698414 +0200
@@ -13,7 +13,7 @@
 # network. Information about each server found is printed to stdout.
 
 # Initialize the environment
-. `dirname $0`/stap-env
+. /usr/share/systemtap/stap-env
 
 #-----------------------------------------------------------------------------
 # Helper functions.
Index: systemtap/stap-gen-cert
===================================================================
--- systemtap.orig/stap-gen-cert	2009-09-10 19:39:11.000000000 +0200
+++ systemtap/stap-gen-cert	2009-09-11 00:19:20.369698414 +0200
@@ -11,7 +11,7 @@
 # later version.
 
 # Initialize the environment
-. `dirname $0`/stap-env
+. /usr/share/systemtap/stap-env
 
 # Obtain a password from stdin and echo it.
 function user_enter_password
Index: systemtap/stap-server
===================================================================
--- systemtap.orig/stap-server	2009-09-10 19:39:11.000000000 +0200
+++ systemtap/stap-server	2009-09-11 00:19:20.369698414 +0200
@@ -17,7 +17,7 @@
 trap 'terminate' SIGTERM SIGINT
 
 # Initialize the environment
-. `dirname $0`/stap-env
+. /usr/share/systemtap/stap-env
 
 #-----------------------------------------------------------------------------
 # Helper functions.
Index: systemtap/stap-serverd
===================================================================
--- systemtap.orig/stap-serverd	2009-09-10 19:39:11.000000000 +0200
+++ systemtap/stap-serverd	2009-09-11 00:19:20.369698414 +0200
@@ -17,7 +17,7 @@
 trap 'terminate' SIGTERM SIGINT
 
 # Initialize the environment
-. `dirname $0`/stap-env
+. /usr/share/systemtap/stap-env
 
 #-----------------------------------------------------------------------------
 # Helper functions.
Index: systemtap/stap-start-server
===================================================================
--- systemtap.orig/stap-start-server	2009-09-10 19:39:11.000000000 +0200
+++ systemtap/stap-start-server	2009-09-11 00:19:20.373026909 +0200
@@ -13,7 +13,7 @@
 # process id, if successful.
 
 # Initialize the environment
-. `dirname $0`/stap-env
+. /usr/share/systemtap/stap-env
 startup_timeout=20
 
 # start the server
